packages/webapp/src/pages/Account/components/Password.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import { Input } from '../../../components/ui/input/Input';
+import { HoverCard, HoverCardContent, HoverCardTrigger, HoverCardArrow } from '../../../components/ui/HoverCard';
+import { Progress } from '../../../components/ui/Progress';
+
+export const Password: React.FC<{ setPassword: (password: string, strength: number) => void } & React.InputHTMLAttributes<HTMLInputElement>> = ({
+    setPassword,
+    ...props
+}) => {
+    const [local, setLocal] = useState('');
+    const { strength, color } = useMemo(() => {
+        let strength = 0;
+        if (local.length >= 1) {
+            strength += 12.5;
+        }
+        if (local.length > 8) {
+            strength += 12.5;
+        }
+        if (local.match(/[A-Z]/)) {
+            strength += 25;
+        }
+        if (local.match(/[0-9]/)) {
+            strength += 25;
+        }
+        if (local.match(/[^a-zA-Z0-9]/)) {
+            strength += 25;
+        }
```

Comment:
I found this confusing when I tested it. 
Even though the bar is supposed to represent the password strength, it doesn't behave like one. For instance a 8 lowercase characters will have the same bar as a password of 32 chars. That doesn't make sense.
I think we either keep the bar but make sure it represents the real strength of the password or we have a check list of requirements that we automatically cross

Reply:
There is not a lot of diff between 8 not random lowercase char and 32 not random lowercase char, since my check is rather naive it's a indication not a scientific check.
Also there is a big diff between 8 random char with numbers and special chars, vs 32 lowercase char (and even bigger vs 32 repeated char for example). 
cf: see here 58minutes for "a"{8} and 4hours for "a"{32}, not really safer while `aAè8-kjn` takes centuries
https://lowe.github.io/tryzxcvbn/


Strength is an indication not something sure, but happy to remove the bar and just show the checks on the right